src/index.html

```diff
@@ -11,7 +11,156 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form>
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="formfield">Surname:
+          <input
+          type="text"
```

Comment:
Please fix for all cases
![image](https://user-images.githubusercontent.com/54936044/222905780-f0c1348f-47c1-470e-8422-b55f9d893f06.png)